src/screens/FeedScreen/index.js

```diff
@@ -92,7 +92,7 @@ const FeedScreen = (props) => {
 
       unsubscribe();
     };
-  }, [navigation]);
+  }, [navigation, route.params.isGoBack]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `route.params.isGoBack` to the dependency array of `useEffect` is a good practice as it ensures that the effect will re-run whenever this value changes. However, please ensure that `route.params.isGoBack` is always defined and has a stable reference to avoid unnecessary re-renders or potential bugs.
```diff
-  }, [navigation, route.params.isGoBack]);
+  }, [navigation, route?.params?.isGoBack]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->